client/src/components/SignUp.js

```diff
@@ -7,7 +7,7 @@ const useStyles = makeStyles((theme)=>({
 login:{
     maxWidth:'1200px',
     margin:'0 auto',
-    height:'100vh',
+    height:'123vh', // Eyeballed to make the form centered in the background. May be a better way to do this.
```

Comment:
As long as it looks ok on a few different monitor sizes, this is fine - you can remove the comment